react/src/pages/utils/components/DetailSection.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { Button, Collapse, Grid, Typography } from "@material-ui/core";
+import { FieldDisplay } from "./components";
+import { FieldDisplayValueType } from "../typings";
+
+const gridSpacing = 2;
+const titleWidth = 12;
+const infoWidth = 6;
+
+interface DetailSectionProps {
+    titles: string[];
+    values: FieldDisplayValueType[];
+    collapsibleTitles?: string[];
+    collapsibleValues?: FieldDisplayValueType[];
+    title?: string;
+}
+export default function DetailSection({
+    titles,
+    values,
+    collapsibleTitles,
+    collapsibleValues,
+    title,
+}: DetailSectionProps) {
+    const [moreDetails, setMoreDetails] = useState(false);
+
+    const getLeftFieldDisplay = (titles: string[], values: FieldDisplayValueType[]) => {
+        return titles.map((title, index) => {
+            if (index >= Math.ceil(titles.length / 2)) {
+                return;
+            } else {
+                return <FieldDisplay title={title} value={values[index]} />;
+            }
+        });
+    };
+    const getRightFieldDisplay = (titles: string[], values: FieldDisplayValueType[]) => {
+        return titles.map((title, index) => {
+            if (index < Math.ceil(titles.length / 2)) {
+                return;
```

Comment:
Same as above comment.
```suggestion
                return (<></>);
```